test/helper.rb

```diff
@@ -138,6 +138,8 @@ def md.directory?
       'test_certificate.rsa.crt.pem' => mockfile.call('test_certificate.rsa.crt.pem'),
       'test_certificate.rsa.key.pem' => mockfile.call('test_certificate.rsa.key.pem'),
       'test_ca_public.key.pem' => mockfile.call('test_ca_public.key.pem'),
+      # Test DH parameters, 2048 bit long safe prime, generator 2 for dh_params in PEM format
+      'dh_params.dh_pem' => mockfile.call('dh_params.dh_pem'),
```

Comment:
Normally I'd add this line in alphabetical order, but the existing recipes are already out of order. Let me know if this requires adjustment.